addons/l10n_it_edi_withholding/__init__.py

```diff
@@ -7,18 +7,11 @@
 
 def _l10n_it_edi_withholding_post_init(env):
     """ Existing companies that have the Italian Chart of Accounts set """
-    template_code = 'it'
-    data = {
-        model: env['account.chart.template']._parse_csv(template_code, model, module='l10n_it_edi_withholding')
-        for model in [
-            'account.account',
-            'account.tax.group',
-            'account.tax',
-        ]
-    }
-    for company in env['res.company'].search([('chart_template', '=', template_code)]):
+    for company in env['res.company'].search([('chart_template', '=', 'it')]):
         _logger.info("Company %s already has the Italian localization installed, updating...", company.name)
-        company_chart_template = env['account.chart.template'].with_company(company)
-        company_data = dict(data)
```

Comment:
It seems like the issue was already detected here, because I think this line fixes the issue you are fixing

I don't mind keeping the forward port, as this solution is cleaner though

Reply:
I had tested it and the issue was present in 17.0.